server/routes/api/gmail-auth.js

```diff
@@ -0,0 +1,156 @@
+const { google } = require("googleapis");
+const mailComposer = require("nodemailer/lib/mail-composer"); // Helps formatting of emails in base64
+const User = require("../../models/user.js");
+const { client_secret, client_id } = require("../../gmail-secret.json");
```

Comment:
I usually don't like relative paths that go backwards - hard to refactor in the future - maybe look up if there is a better way to do this online?